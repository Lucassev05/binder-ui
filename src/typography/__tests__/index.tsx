import React from "react";
import "@testing-library/jest-dom/extend-expect";
import { render } from "../../utils/render";
import { Span } from "..";

describe("<Span /> test case", () => {
  test("renders text", () => {
    const { getByText } = render(<Span>Hi, I'm Span</Span>);
    expect(getByText("Hi, I'm Span")).toBeInTheDocument();
  });

  test("matches snapshot", () => {
    const { getByText } = render(<Span>Hi, I'm Span</Span>);
    expect(getByText("Hi, I'm H4")).toMatchSnapshot();
  });
});
