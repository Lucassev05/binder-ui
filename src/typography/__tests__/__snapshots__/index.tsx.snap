// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Span /> test case matches snapshot 1`] = `
.c0 {
  font-family: IBM Plex Sans,sans-serif;
  font-style: normal;
  -webkit-letter-spacing: 0.6684448px;
  -moz-letter-spacing: 0.6684448px;
  -ms-letter-spacing: 0.6684448px;
  letter-spacing: 0.6684448px;
  color: #202020;
}

<span
  class="c0"
>
  Hi, I'm Span
</span>
`;
